unattended.sh

```diff
@@ -694,6 +686,59 @@ function update_after_installation() {
 }
 #========= end of function update_after_installation()
 
+#========= begin of function enable_debug_mode()
+function enable_debug_mode() {
+	log "INFO" "Enabling advanced logging and debug options"
+	log "INFO" "Attention: this option will create many logs"
+
+	if [ "$wizard_autoplay" == "true" ]; then
+		mysql -u root centreon <<-EOF
+			DELETE FROM options WHERE options.key IN ('debug_path','debug_auth','debug_nagios_import','debug_rrdtool','debug_ldap_import','debug_sql','debug_gorgone','debug_centreontrapd');
+			INSERT INTO options VALUES ('debug_path','/var/log/centreon/');
+			INSERT INTO options VALUES ('debug_auth','1');
+			INSERT INTO options VALUES ('debug_nagios_import','1');
+			INSERT INTO options VALUES ('debug_rrdtool','1');
+			INSERT INTO options VALUES ('debug_ldap_import','1');
+			INSERT INTO options VALUES ('debug_sql','1');
+			INSERT INTO options VALUES ('debug_gorgone','1');
+			INSERT INTO options VALUES ('debug_centreontrapd','1');
+EOF
+	fi
+
+	mkdir -p /var/log/mysql
+	chown mysql:mysql /var/log/mysql
+	printf "\n[mariadb]\nlog_output=FILE\ngeneral_log\ngeneral_log_file=/var/log/mysql/mariadb.log\nlog_error\nlog_error=/var/log/mysql/mariadb.err\n" >> /etc/my.cnf.d/custom_log.cnf
```

Comment:
```suggestion
       cat <<EOF >> /etc/my.cnf.d/custom_log.cnf

[mariadb]
log_output=FILE
general_log
general_log_file=/var/log/mysql/mariadb.log
log_error\nlog_error=/var/log/mysql/mariadb.err

EOF
```
Wouldn't using EOF get more intuitive and visual?